Mathlib/Analysis/Complex/AbelLimit.lean

```diff
@@ -0,0 +1,212 @@
+/-
+Copyright (c) 2024 Jeremy Tan. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Jeremy Tan
+-/
+import Mathlib.Analysis.Complex.Basic
+import Mathlib.Analysis.SpecificLimits.Normed
+
+/-!
+# Abel's limit theorem
+
+If a real or complex power series for a function has radius of convergence 1 and the series is only
+known to converge conditionally at 1, Abel's limit theorem gives the value at 1 as the limit of the
+function at 1 from the left. "Left" for complex numbers means "within a Stolz set", a subset of the
+open unit disc with 1 on its boundary.
```

Comment:
Is there a reason why you use Stolz sets instead of the simpler cone of angle \le \theta < pi/2 based at 1? This would be more geometric and easier to use in practice.

Reply:
IIRC, the condition comes up naturally in the proof. One could add a variant using the cone after the fact, though.

Reply:
@Parcly-Taxel can you address this comment before you mark `awaiting-review` again?

Reply:
@j-loreaux I don't know how to do it.

Reply:
@sgouezel I don't know how to change to the cone filter without importing all of Euclidean geometry. There's no simple way to do it.

Reply:
@j-loreaux yeah, done.